variables.tf

```diff
@@ -273,7 +273,7 @@ variable "log_retention_in_days" {
 
 variable "microsoft_defender_enabled" {
   type        = bool
-  description = "(Optional) Is Microsoft Defender on the cluster enabled?"
+  description = "(Optional) Is Microsoft Defender on the cluster enabled? Requires a Log Analytics Workspace."
```

Comment:
Could the description be more specific? Like: 

```hcl
description = "(Optional) Is Microsoft Defender on the cluster enabled? Requires `var.log_analytics_workspace_enabled` to be `true` to set this variable to `true`."
```

Reply:
I agree. This is the way.